politeiawww/cmswww.go

```diff
@@ -131,6 +131,68 @@ func (p *politeiawww) handleInvoiceDetails(w http.ResponseWriter, r *http.Reques
 	util.RespondWithJSON(w, http.StatusOK, reply)
 }
 
+// handleSetInvoiceStatus handles the incoming set invoice status command.
+func (p *politeiawww) handleSetInvoiceStatus(w http.ResponseWriter, r *http.Request) {
+	// Add the path param to the struct.
+	log.Tracef("handleSetInvoiceStatus")
+	var sis cms.SetInvoiceStatus
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&sis); err != nil {
+		RespondWithError(w, r, 0, "handleSetInvoiceStatus: unmarshal", www.UserError{
+			ErrorCode: www.ErrorStatusInvalidInput,
+		})
+		return
+	}
+
+	user, err := p.getSessionUser(w, r)
+	if err != nil {
+		RespondWithError(w, r, 0,
+			"handleSetInvoiceStatus: getSessionUser %v", err)
+		return
+	}
+
+	reply, err := p.processSetInvoiceStatus(sis, user)
+	if err != nil {
+		RespondWithError(w, r, 0,
+			"handleSetInvoiceStatus: processSetInvoiceStatus %v", err)
+		return
+	}
+
+	// Reply with the challenge response and censorship token.
+	util.RespondWithJSON(w, http.StatusOK, reply)
+}
+
+// handleEditInvoice attempts to edit an invoice
+func (p *politeiawww) handleEditInvoice(w http.ResponseWriter, r *http.Request) {
+	var ei cms.EditInvoice
```

Comment:
Pi repo convention is to have a `log.Tracef` for these handle and process functions.